common/python/schema_validation/WorkOrderSubmit.py

```diff
@@ -0,0 +1,224 @@
+                            "pattern": "(0[x|X])?[0-9a-fA-F]",
+                            "error_msg":
+                            "Invalid data format for \
+data hash of in data"},
+                        "encryptedDataEncryptionKey": {
+                            "type": "string",
+                            "pattern": "^-$|^$|^null$|^(0[x|X])?[0-9a-fA-F]+$",
+                            "error_msg":
+                            "Invalid data format for \
+Encryption key of in data"},
+                        "iv": {
+                            "type": "string",
+                            "pattern": "^$|^(0[x|X])?[0-9a-fA-F]+$",
+                            "error_msg":
+                            "Invalid data format for \
+initialization vector of in data"}
+                    },
+                    "required": ["index", "data"]
+                }
+            },
+            "outData": {
+                "type": "array",
+                "uniqueItems": True,
+                "additionalProperties": False,
+                "items": {
+                    "type": "object",
+                    "error_msg":
+                    "Invalid data format for outData ",
+                    "properties": {
+                        "index": {
+                            "type": "integer",
+                            "error_msg":
+                            "Invalid data format for outData index"},
+                        "dataHash": {
+                            "type": "string",
+                            "pattern": "(0[x|X])?[0-9a-fA-F]",
+                            "error_msg":
+                            "Invalid data format for \
+data hash of out data"},
+                        "data": {
+                            "type": "string",
+                            "contentEncoding": "base64",
+                            "error_msg":
+                            "Invalid data format for out data"},
+                        "encryptedDataEncryptionKey": {
+                            "type": "string",
+                            "pattern": "^-$|^$|^null$|^(0[x|X])?[0-9a-fA-F]+$",
+                            "error_msg":
+                            "Invalid data format for \
+Encryption key of out data"},
+                        "iv": {
+                            "type": "string",
+                            "pattern": "^$|^(0[x|X])?[0-9a-fA-F]+$",
+                            "error_msg":
+                            "Invalid data format for \
+initialization vector of out data"}
+                    }
+                }
+            },
+        },
+        "required": [
+            "responseTimeoutMSecs",
+            "payloadFormat",
+            "workOrderId",
+            "workerId",
+            "workloadId",
+            "requesterId",
+            "encryptedSessionKey",
+            "requesterNonce",
+            "encryptedRequestHash",
+            "inData"
+            ]
+    }
```

Comment:
Please move above schema definition to a file

Reply:
done